components/reaction-badge.json

```diff
@@ -0,0 +1,34 @@
+{
+  "reaction-badge": {
+    "comment": "Applied to reaction buttons",
+    "figma": "https://www.figma.com/file/NaNrfXjygZtRgMfHAFHjsp/Components---Windows%2BWeb?node-id=5585%3A13103",
+    "primary": {
+      "#normal": {
+        "background": "@theme-background-secondary-normal",
+        "text": "@theme-text-primary-normal",
+        "border": "@theme-outline-secondary-normal"
+      },
+      "#hovered": {
+        "background": "@theme-background-secondary-hover",
+        "text": "@theme-text-primary-normal",
```

Comment:
When values are the same over different UI states, we don't need to bother including them.